theme_nano/views/snippets/s_references_social.xml

```diff
@@ -0,0 +1,62 @@
+<?xml version="1.0" encoding="utf-8"?>
+<odoo>
+
+<template id="s_references_social" inherit_id="website.s_references_social">
+    <!-- Section -->
+    <xpath expr="//div[hasclass('row')]/div[2]/h3" position="replace" mode="inner">
+        Barbershop
+    </xpath>
+    <xpath expr="//div[hasclass('row')]/div[3]/h3" position="replace" mode="inner">
+        Oceandor
+    </xpath>
+    <xpath expr="//div[hasclass('row')]/div[4]/h3" position="replace" mode="inner">
+        Mountain
+    </xpath>
+    <xpath expr="//div[hasclass('row')]/div[5]/h3" position="replace" mode="inner">
+        Hosoren
+    </xpath>
+
+    <!-- s_social color better contrast -->
```

Comment:
Is it really needed ? I think the contrast is even better without 

with this change:
<img width="190" alt="Screenshot 2024-09-09 at 14 25 21" src="https://github.com/user-attachments/assets/8a1480ed-ed1a-45dd-9265-8e3b62b044d3">

default:
<img width="185" alt="Screenshot 2024-09-09 at 14 24 17" src="https://github.com/user-attachments/assets/9a4b7b3e-af84-4725-9a9e-b73b6e3b85ee">